content/en/docs/concepts/security/security-checklist.md

```diff
@@ -0,0 +1,153 @@
+---
+reviewers:
+title: Security Checklist
+description: >
+  Recommended checklist for ensuring security in Kubernetes clusters.
+content_type: concept
+---
+
+<!-- overview -->
+
+This checklist aims at providing a basic list of guidance with links to more
+comprehensive documentation on each topic. It does not claim to be exhaustive
+and is meant to evolve.
+
+The order of topics does not reflect an order of priority.
+
+<!-- body -->
+
+## Authentication & Authorization
+
+See the [Role Based Access Control Good Practices](/docs/concepts/security/rbac-good-practices/)
+for information related to authentication and authorization.
+
+<!-- New authentication certificates should be generated on new clusters that avoid -->
+<!-- the use of the `system:masters`. `system:masters` should be used as a -->
+<!-- break-glass mechanism as opposed to an admin user. -->
+
+## Network security
+
+A number of Container Network Interfaces (CNIs) provide functionality to
```

Comment:
```suggestion
A number of [Container Network Interfaces](https://github.com/containernetworking/cni) (CNIs) provide functionality to
```

Reply:
```suggestion
A number of [Container Network Interface (CNI) plugins](/docs/concepts/extend-kubernetes/compute-storage-net/network-plugins/)
plugins provide the functionality to
```

Reply:
Great idea for the link, I didn't notice the confusion between the CNI and CNI plugins that were made in the checklist. So I fixed this and put the link to the network plugins doc, that features a link to the CNI, that's an indirection!